src/Phinx/Db/Table.php

```diff
@@ -541,6 +566,21 @@ public function addTimestamps()
     }
 
     /**
+     * @param $data array of data in the form :
+     *              array(
+     *                  array("column1" => "value1", "column2" => "anotherValue1"),
+     *                  array("column1" => "value2", "column2" => "anotherValue2"),
+     *              )
+     *
+     * @return Table
+     */
+    public function insert($data)
+    {
+        $this->data = $data;
```

Comment:
This doesn't really make sense to me... INSERTs have 3 parts:

- a table name
- **one** set of columns
- one **or more** rows

I don't think it is good practice to force the insert data to contain the column names in every line.